static/src/javascripts/projects/commercial/modules/messenger/background.js

```diff
@@ -93,7 +101,27 @@ const setBackground = (specs: AdSpec, adSlot: Node): Promise<any> => {
         return fastdom
             .write(() => {
                 if (backgroundParent) {
-                    adSlot.insertBefore(backgroundParent, adSlot.firstChild);
+                    if (specs.scrollType === 'interscroller') {
+                        adSlot.style.height = '85vh';
+                        // $FlowFixMe
+                        adSlot.style['margin-bottom'] = '12px';
```

Comment:
You could write `adSlot.style.marginBottom` here instead of `adSlot.style['margin-bottom']`.